proto/prysm/storage/BUILD.bazel

```diff
@@ -0,0 +1,33 @@
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
+load("@prysm//tools/go:def.bzl", "go_library")
+
+# gazelle:ignore
+proto_library(
+    name = "proto",
+    srcs = [
+        "beacon_storage.proto",
+        "version.proto",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        "//proto/eth/v2:proto",
+    ],
+)
+
+go_proto_library(
+    name = "go_proto",
+    importpath = "github.com/prysmaticlabs/prysm/proto/prysm/storage",
+    proto = ":proto",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//proto/eth/v2:go_default_library",
```

Comment:
I don't see this dep being used in code. Do you need to re-run gazelle?

Reply:
Odd, thought the build would have failed in that case.

Reply:
gazel doesn't properly handle this proto deps. It was handcrafted.
oops. it got bulldozed and merged. I will remove it in some other PR.